viewer/src/main/webapp/viewer-html/components/DownloadWKT-config.js

```diff
@@ -0,0 +1,129 @@
+/* 
+ * Copyright (C) 2018 B3Partners B.V.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+Ext.define("viewer.components.CustomConfiguration",{
+    extend: "viewer.components.SelectionWindowConfig",
+     constructor: function (parentId, configObject, configPage) {
+        if (configObject === null){
+            configObject = {};
+        }
+        viewer.components.CustomConfiguration.superclass.constructor.call(this, parentId, configObject, configPage);
+        this.addForm(configObject);
+        var types = configObject.types;
+        if(types){
+            for (var i = 0; i < types.length; i++) {
+                this.createRow(types[i]);
+            }
+        }
+    },
+    addForm: function() {
+        this.form.add({ 
+                xtype: 'textfield',
+                fieldLabel: 'Basispad *',
+                name: 'basePath',
```

Comment:
I think this should default to the flamingo data dir or the java.io.tmpdir, perhaps it it should not even be configurable from the viewer-admin at all.

Reply:
File must be saved to disk, so an automated process can pick it up for processing. Asked if it was possible for that folder to be in flamingo_data_dir.